internal/store/atlas/projects.go

```diff
@@ -64,56 +65,67 @@ type ProjectTeamDeleter interface {
 
 // Projects encapsulates the logic to manage different cloud providers.
 func (s *Store) Projects(opts *atlas.ListOptions) (interface{}, error) {
-	result, _, err := s.client.Projects.GetAllProjects(s.ctx, opts)
+	result, _, err := s.clientv2.ProjectsApi.ListProjects(s.ctx).PageNum(int32(opts.PageNum)).
+		ItemsPerPage(int32(opts.ItemsPerPage)).IncludeCount(opts.IncludeCount).Execute()
 	return result, err
 }
 
 // GetOrgProjects encapsulates the logic to manage different cloud providers.
 func (s *Store) GetOrgProjects(orgID string, opts *atlas.ProjectsListOptions) (interface{}, error) {
-	result, _, err := s.client.Organizations.Projects(s.ctx, orgID, opts)
+	result, _, err := s.clientv2.OrganizationsApi.ListOrganizationProjects(s.ctx, orgID).IncludeCount(opts.IncludeCount).
+		PageNum(int32(opts.PageNum)).Name(opts.Name).ItemsPerPage(int32(opts.ItemsPerPage)).Execute()
 	return result, err
 }
 
 // Project encapsulates the logic to manage different cloud providers.
 func (s *Store) Project(id string) (interface{}, error) {
-	result, _, err := s.client.Projects.GetOneProject(s.ctx, id)
+	result, _, err := s.clientv2.ProjectsApi.GetProject(s.ctx, id).Execute()
 	return result, err
 }
 
 // CreateProject encapsulates the logic to manage different cloud providers.
-func (s *Store) CreateProject(name, orgID, regionUsageRestrictions string, defaultAlertSettings *bool, opts *atlas.CreateProjectOptions) (interface{}, error) {
-	project := &atlas.Project{Name: name, OrgID: orgID, RegionUsageRestrictions: regionUsageRestrictions, WithDefaultAlertsSettings: defaultAlertSettings}
```

Comment:
@gssbzn @wtrocki

RegionUsageRestrictions is [undocumented on the API](https://github.com/10gen/mms/blob/master/server/src/main/com/xgen/svc/mms/api/view/atlas/ApiAtlasGroupView.java#L104-L109)

I've raised this PR https://github.com/10gen/mms/pull/72618 to make the property public, so if it's required we can go with that and update this PR once merged 

Otherwise, can go with this as is